libs/backend/src/routes/puzzle/[id]/index.ts

```diff
@@ -90,6 +92,33 @@ export default async function puzzleByIdRoutes(fastify: FastifyInstance) {
 
 				await puzzle.save();
 
+				const checkWhenEdited: PuzzleVisibility[] = [
+					puzzleVisibilityEnum.DRAFT,
+					puzzleVisibilityEnum.READY,
+					puzzleVisibilityEnum.REVIEW
+				];
+
+				if (
+					checkWhenEdited.includes(puzzle.visibility) &&
+					puzzle.validators &&
+					puzzle.validators.length > 0 &&
```

Comment:
I think we should require at least a few test cases before marking it ready. On CodinGame you need 8 (4 tests and 4 validators) and even that feels like it's usually not enough. Maybe something like 6 would be a reasonable starting point.

Reply:
i made it 10, rather too many than too few :smile: